packages/create-block/lib/templates.js

```diff
@@ -77,11 +44,25 @@ const getTemplate = ( templateName ) => {
 };
 
 const getDefaultValues = ( templateName ) => {
-	return getTemplate( templateName ).defaultValues;
+	return {
+		namespace: 'create-block',
+		dashicon: 'smiley',
+		category: 'widgets',
+		author: 'The WordPress Contributors',
+		license: 'GPL-2.0-or-later',
+		licenseURI: 'https://www.gnu.org/licenses/gpl-2.0.html',
+		version: '0.1.0',
+		...getTemplate( templateName ).defaultValues,
+	};
 };
 
-const getOutputFiles = ( templateName ) => {
-	return getTemplate( templateName ).outputFiles;
+const getOutputFiles = async ( templateName ) => {
+	const templatesPath = getTemplate( templateName ).templatesPath;
+
+	return await glob( '**/*.mustache', {
+		cwd: join( __dirname, templatesPath ),
```

Comment:
@fabiankaegy, I guess we will have to merge your changes first and then figure out how to handle the path for templates downloaded from npm.